backend/stream.go

```diff
@@ -75,6 +75,13 @@ func NewInitialFrame(frame *data.Frame, include data.FrameInclude) (*InitialData
 	}, nil
 }
 
+// NewInitialData allows sending bytes on subscription
+func NewInitialData(data []byte) (*InitialData, error) {
+	return &InitialData{
```

Comment:
Maybe also check for valid JSON here like we already do [here](https://github.com/grafana/grafana-plugin-sdk-go/blob/master/backend/stream.go#L144) ?

Reply:
yup

Reply:
why not use `json.RawMessage` instead of `[]byte` :eyes: